src/pages/sponsors.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import { Helmet } from "react-helmet"
+import styles from "../styles/sponsors.module.css"
+import { graphql, Link } from 'gatsby'
+
+const SponsorsPage = ( {data} ) => {
+
+	const getImageUrl = (companyName) => {
+		const URLRegex = /((([A-Za-z]{3,9}:(?:\/\/)?)(?:[\-;:&=\+\$,\w]+@)?[A-Za-z0-9\.\-]+|(?:www\.|[\-;:&=\+\$,\w]+@)[A-Za-z0-9\.\-]+)((?:\/[\+~%\/\.\w\-_]*)?\??(?:[\-\+=&;%@\.\w_]*)#?(?:[\.\!\/\\\w]*))?)/ ;
+		return URLRegex.test(companyName) ? companyName : 'https://res.cloudinary.com/chicagojs/image/upload/2018Sponsors/' + companyName;
+	}
+
+	const grabTieredSponsors = (tier) => {
+		const sponsorsArray = data.allSponsorsDataJson.edges.find(data => data.node.title === tier);
+		if(sponsorsArray.node.items.length > 0) {
+			return sponsorsArray.node.items.map((i,index) => (
+				<div className={`col-md-6 col-lg-3 ${styles.sponsorCard} ${tier === 'gold' ? 'col-lg-4' : 'col-lg-2'}`}>
```

Comment:
I like using the `classNames` library for conditional classes just as it really helps readability.